mock-core-configs/etc/mock/templates/epel-8.tpl

```diff
@@ -17,8 +17,12 @@ mirrorlist=http://mirrors.fedoraproject.org/mirrorlist?repo=testing-epel8&arch=$
 failovermethod=priority
 skip_if_unavailable=False
 
+{% if koji_primary_repo != None and koji_primary_repo != "epel" %}
+[local-epel]
+{% else %}
 [local]
-name=local
+{% endif %}
+name=Extra Packages for Enterprise Linux $releasever - Koji Local WARNING! FOR BUILDROOT ONLY!
 baseurl=https://kojipkgs.fedoraproject.org/repos/epel8-build/latest/$basearch/
 cost=2000
 enabled=0
```

Comment:
Can you move this down to the bottom like how epel-9.tpl has it?

Reply:
Done, in separate commit.